backend/src/backend/main.py

```diff
@@ -95,20 +97,25 @@ async def add_book(book_payload: BookAddSchema):
                 kind=KIND_TEXT,
             )
 
+    title = await Title.objects.get_or_create(
+        ident=get_ident_from_name(book_payload.metadata["Name"])
+    )
+    await book.update(title=title)
+
     for tag_name in book_payload.metadata["Tags"].split(";"):
         book_tag = await BookTag.objects.get_or_create(name=tag_name)
         await book.tags.add(book_tag)
+        await title.tags.add(book_tag)
```

Comment:
We want only the non-book specific Tags.

Book-specific Tags to exclude:
- `_sw:<no|yes>`
- `_ftindex:<no|yes>`
- `_pictures:<no|yes>`
- `_videos:<no|yes>`
- `_details:<no|yes>`

So it's mostly the *private* tags (starting with an `_`) but the `_category:xxx` one. Please specifically exclude the list above.